src/core/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "scripts": {
     "lint": "./node_modules/.bin/tslint **/*.ts --exclude node_modules/**",
-    "test": "./node_modules/.bin/ts-mocha **/*.spec.ts",
+    "test": "./node_modules/.bin/ts-mocha '**/*.spec.ts'",
```

Comment:
A little bit out of the scope of this PR, but since the tests were not working locally without the quotes I went ahead and fixed it as part of the same PR.

It's a good practice to quote globs in scripts for increased portability.